prog/vm/nexus.rb

```diff
@@ -425,6 +448,10 @@ def write_params_json
     nap 30
   end
 
+  label def prep_failed
+    nap(5 * 60)
```

Comment:
it can be much longer. A regression from system 3: naps were often 1000 years (I did that so the first digit was notable in dumps, but you got the lower bits to intuit what time the nap was done). _but_ if you look are the semaphore code, it wakes up a strand when it happens, that's how the whole system avoided polling. That this one does not do the same is vestigial/extraneous habits from system 2.

Reply:
I'll add a PR to update the nap time after merging this.